app/models/record.rb

```diff
@@ -3,8 +3,15 @@ class Record < ApplicationRecord
 
   include SearchScope
   belongs_to :register
-  scope :current, -> { where(Arel.sql("data->> 'end-date' is null")) }
-  scope :archived, -> { where(Arel.sql("data->> 'end-date' is not null")) }
+  scope :current, -> {
+    where(Arel.sql("data->> 'end-date' is null or data->> 'end-date' >= '#{Date.today}'"))
+  }
+
+  scope :archived, -> {
+    # where("data->> 'end-date' < ?", Date.today)
```

Comment:
I think this version is preferable rather than interpolating within the SQL.

Reply:
This field is actually denormalised directly on the record so we should be able to do:
`where('end_date < ?', Date.today)` with equivalence

Reply:
Super - I've switched to the Rails magic version. There are another couple of queries using `Arel.sql` in this model - but I think that's a non-urgent fix for another day.

Reply:
Yes, we have to use `Arel.sql` to query JSONB as it is not part of standard SQL.